src/contexts/pageContext.tsx

```diff
@@ -15,17 +15,15 @@ const defaultFlashProps: FlashProps = {
 
 const PageContext = createContext<PageContextType>({
   flashProps: defaultFlashProps,
-  setFlashProps(this: PageContextType, props: FlashProps) {
-    this.flashProps = props
-  },
+  setFlashProps: () => {},
 })
 
 const PageProvider = ({ children }: ProviderProps) => {
   const [flashProps, setFlashProps] = useState<FlashProps>(defaultFlashProps)
 
   useEffect(() => {
     if (flashProps.hidden === false) {
-      setTimeout(() => setFlashProps({ ...flashProps, hidden: true }), 3000)
+      setTimeout(() => setFlashProps({ ...flashProps, hidden: true }), 4000)
```

Comment:
This needed to increase since it took me about 4 seconds to read the longer messages.